Modules/TPC/src/CheckOfPads.cxx

```diff
@@ -0,0 +1,337 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   CheckOfPads.cxx
+/// \author Laura Serksnyte, Maximilian Horst
+///
+
+#include "TPC/CheckOfPads.h"
+#include "QualityControl/MonitorObject.h"
+#include "QualityControl/Quality.h"
+#include "QualityControl/QcInfoLogger.h"
+#include "Common/Utils.h"
+
+// ROOT
+#include <TCanvas.h>
+#include <TH1.h>
+#include <TH2.h>
+#include <TPad.h>
+#include <TList.h>
+#include <TPaveText.h>
+
+#include <iostream>
+
+namespace o2::quality_control_modules::tpc
+{
+void CheckOfPads::configure()
+{
+  mMediumQualityLimit = common::getFromConfig<float>(mCustomParameters, "mediumQualityPercentageOfWorkingPads", 0.7);
+  mBadQualityLimit = common::getFromConfig<float>(mCustomParameters, "badQualityPercentageOfWorkingPads", 0.3);
+  std::string CheckChoiceString = common::getFromConfig<std::string>(mCustomParameters, "CheckChoice", "Mean");
```

Comment:
Naming convention and logic
```suggestion
  std::string checkChoiceString = common::getFromConfig<std::string>(mCustomParameters, "CheckChoice", "Mean");
```
If the default value is `Mean`, the `mFoundCheck` will always be true. Is this the intended behaviour? If not, I think you should simple remove the `, "Mean"` part. Otherwise you can remove the lines 53-56 below, because this part of the code will never be executed...

Also `mFoundCheck` only seems to be used in this function, so no need for a member variable... Depending on the question above, it might even be eliminated completely.

Reply:
what about `const` and naming convention?